tinc.yaml

```diff
@@ -0,0 +1,3 @@
+dependencies:
+  - name: riak-protobuf
+    path: protobuf
```

Comment:
It does caching in a similar way to tinc from what I can see.

If you want to try it out drop the [mafia](https://github.com/ambiata/mafia/blob/master/script/mafia) script into the top level of a project, make it executable and go from there.